ngraph/frontend/paddlepaddle/src/op/layer_norm.cpp

```diff
@@ -0,0 +1,58 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <ngraph/opsets/opset6.hpp>
```

Comment:
I am thinking of a common header for paddle/op folder, to include opsetXX.hpp, and namespace, how do you think?

Reply:
Added default_opset.hpp like ONNX mode. I think we may need another PR to sync the modification if it's acceptable.

Reply:
great. Let's do it after our bert PRs merged. :)